app/controllers/admin/settings_controller.rb

```diff
@@ -26,10 +26,10 @@ def update_all
 
   private
 
-  def update_attributes(setting_params)
+  def update(setting_params)
```

Comment:
don't call this method just update ....
update is usually used as an controller action for PATCH update.
renamed it to update_settings
no need to add params here ... params[:setting] is available in the whole controller